pkg/nrtupdater/nrtupdater_test.go

```diff
@@ -51,8 +52,8 @@ func TestUpdateTMPolicy(t *testing.T) {
 
 	var err error
 	k8sClient := clientk8sfake.NewSimpleClientset()
-	nodeGetter := OneshotNodeGetter{K8SCli: k8sClient}
-	nrtUpd = NewNRTUpdater(&nodeGetter, args, tmConfInitial)
+	nodeGetter := NewCachedNodeGetter(k8sClient, context.TODO())
```

Comment:
`context.Background()` has the same behavior but conveys better semantic: we're fine with unbounded wait (no cancelation), and this is what we want (vs TODO which is meant to be a temporary placeholder, but oftentimes abused as synonym for Background)